molecule/yaml_config_validation/requirements.yml

```diff
@@ -0,0 +1,5 @@
+---
+collections:
```

Comment:
@RanabirChakraborty in a separate commit, you should move the requirement file from molecule/default to just molecule and adapt both molecule/*/molecule.yml to use it (so avoid duplicating configuration).

Reply:
Done, Thanks